src/DiscordServer.js

```diff
@@ -253,8 +253,8 @@ class DiscordServer {
 
         let rank = 0
         for (const groupObj of groups) {
```

Comment:
Since changing to the new API, this throws the error
```
TypeError: groups is not iterable
at Function.resolveGroupRankBinding (/app/src/DiscordServer.js:255:32)
    at processTicksAndRejections (internal/process/task_queues.js:88:5)
    at async Promise.all (index 0)
```

meaning that groups.data isn't the correct format...

Reply:
Resolved with review fixes.